modules/vault/examples_test.go

```diff
@@ -34,3 +35,76 @@ func ExampleRunContainer() {
 	// Output:
 	// true
 }
+
+func ExampleRunContainer_withToken() {
```

Comment:
@kuisathaverat I fixed the testable example name, as it does not find a `RunContainerWithToken` function in the code. The testable examples need to follow this pattern: `Example${FunctionName}`. Please see https://go.dev/blog/examples#example-function-names